sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md

```diff
@@ -254,4 +254,4 @@ If you'd like to contribute to this library, please read the [contributing guide
 
 ### License
 
-Azure SDK for Embedded C is licensed under the [MIT][azure_sdk_for_c_license] license.
\ No newline at end of file
+Azure SDK for Embedded C is licensed under the [MIT][azure_sdk_for_c_license] license.
```

Comment:
What's the change here?

Reply:
it's a ghost character!

Reply:
I am wondering the same

Reply:
Is it a line-ending changed from CRLF to LF or vice versa? You can use hexdump to see the exact byte contents of the file and that will show you the difference:
https://www.fileformat.info/tool/hexdump.htm

Reply:
I bet it's a new line at the end of the file.

Reply:
"> \ No newline at end of file
+Azure SDK for Embedded C is licensed under the [MIT][azure_sdk_for_c_license] license.
"

Was in the email.

Reply:
except now this is all obsolete because I just merged in a change to this same file... Hopefully it's not much extra work for you!